_pkgdown.yml

```diff
@@ -1,37 +1,29 @@
+# Colors should stay consistent across epipredict & epidatr, using Carnegie
+# Red https://www.cmu.edu/brand/brand-guidelines/visual-identity/colors.html
 template:
   bootstrap: 5
   bootswatch: cosmo
   bslib:
     font_scale: 1.0
-    primary: '#C41230'
-    link-color: '#C41230'
-    navbar-bg: '#C41230'
-    navbar-fg: '#f8f8f8'
-
-development:
-  mode: auto
+    primary: "#C41230"
+    link-color: "#C41230"
 
 navbar:
-  bg: '#C41230'
-  fg: '#f8f8f8'
+  bg: primary
+  type: dark
```

Comment:
doesn't seem to match epiprocess [here](https://github.com/cmu-delphi/epiprocess/blob/fd2339d8379455fda1ff27cc078fb23be5c3a93c/_pkgdown.yml#L7).  Are they actually the same or do we need to update one of them?

Reply:
I think we need to update epiprocess because its [docs don't have the right colors either](https://cmu-delphi.github.io/epiprocess/dev/) (though they don't have the strange invisibility problem).